### PR TITLE
Bump julia version and add JuliaFormatter

### PR DESCRIPTION
**Pull request by mvanzulli**
Closes: #46, #47

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.6']
+        julia-version: ['1']
```
Comment (mvanzulli) [inline .github/workflows/ci.yml:22]: For context: https://github.com/JuliaLang/Example.jl/pull/68#discussion_r995947019

File: examples/example_9_1_Bathe.jl [hunk 2]
```diff
@@ -1,60 +1,57 @@
 # # Example (Ch. 9 Bathe)
 
-#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/models/example_9_1_Bathe.ipynb)
```
Comment (mvanzulli) [inline examples/example_9_1_Bathe.jl:3]: This is throwing some errors, I don't find value to fix it right now. If you are keen feel free to tackle it or we can create a good first issue to migrate to Pluto and deploy those directly.

**Comment by mvanzulli** [comment]
Locally works ok to me: 

image

Will explore to set again the Documenter CI keys for CD

**Comment by mvanzulli** [comment]
@mforets Any idea for this? Otherwise I would merge and address the docs pages deployment as a follow-up

**Review by mforets — APPROVED** [review]

**Comment by mforets** [comment]
sounds good, thanks for adding the formatter.

**Pull request merged**